### ISSUE TITLE
2019-12-22 tc013 M2SD, tc014 M2SS Estimated Concentration Dataset not sorted in SDEID and TIME order, CEST_C0 at TIME==0 missing

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#136]
As of commit b9591d0, with parameterset either "PARAMETERLIST" or "PARAMETERDISPLAYLIST" or mct$TIME set to either "NOMINAL" or "ACTUAL" some records are duplicated and the remainder are not sorted in TIME order by SDEID.

Note also that the value for CEST_C0 at TIME==0 is missing.

example Estimated Concentration Dataset snippet:

```r
> e
     PKDATAROWID     SDEID   TIME  CEST_KEL  CEST_INT     CEST_EXT CEST_C0 CEST_TLAST
1             NA  10589286  20.00        NA 142.00000           NA      NA         NA
2       32321091  10589286  72.00  37.59058        NA           NA      NA         NA
3       32321092  10589286  96.00  28.42571        NA           NA      NA         NA
4       32321093  10589286 120.00  21.49531        NA           NA      NA         NA
5       32321094  10589286 168.00  12.29161        NA           NA      NA   12.29161
6             NA  10589286 696.00        NA        NA 2.627372e-02      NA         NA
7             NA  10589286   0.25        NA        NA           NA   332.0         NA
8             NA  10589286   0.50        NA        NA           NA   295.0         NA
11            NA  10589286  20.00        NA 142.00000           NA      NA         NA
21      32321091  10589286  72.00  37.59058        NA           NA      NA         NA
31      32321092  10589286  96.00  28.42571        NA           NA      NA         NA
41      32321093  10589286 120.00  21.49531        NA           NA      NA         NA
51      32321094  10589286 168.00  12.29161        NA           NA      NA   12.29161
61            NA  10589286 696.00        NA        NA 2.627372e-02      NA         NA
17      32321189  45060990   0.50 352.95353        NA           NA      NA         NA
22      32321190  45060990   1.00 350.31335        NA           NA      NA         NA
32      32321191  45060990   1.50 347.69292        NA           NA      NA         NA
42      32321192  45060990   2.00 345.09210        NA           NA      NA         NA
52      32321193  45060990   3.00 339.94866        NA           NA      NA         NA
62      32321194  45060990   5.00 329.89063        NA           NA      NA         NA
71      32321195  45060990   8.00 315.35880        NA           NA      NA         NA
81      32321196  45060990  12.00 296.97387        NA           NA      NA         NA
15            NA  45060990  20.00        NA 199.66667           NA      NA         NA
9       32321197  45060990  24.00 248.00367        NA           NA      NA         NA
10      32321198  45060990  48.00 172.95689        NA           NA      NA         NA
111     32321199  45060990  72.00 120.61952        NA           NA      NA         NA
12      32321200  45060990  96.00  84.11963        NA           NA      NA         NA
13      32321201  45060990 120.00  58.66473        NA           NA      NA         NA
14      32321202  45060990 168.00  28.53229        NA           NA      NA   28.53229
16            NA  45060990 696.00        NA        NA 1.027758e-02      NA         NA
171           NA  45060990   0.25        NA        NA           NA   485.0         NA
19      32321189  45060990   0.50 352.95353        NA           NA      NA         NA
18            NA  45060990   0.50        NA        NA           NA   467.0         NA
23      32321190  45060990   1.00 350.31335        NA           NA      NA         NA
33      32321191  45060990   1.50 347.69292        NA           NA      NA         NA
43      32321192  45060990   2.00 345.09210        NA           NA      NA         NA
53      32321193  45060990   3.00 339.94866        NA           NA      NA         NA
63      32321194  45060990   5.00 329.89063        NA           NA      NA         NA
72      32321195  45060990   8.00 315.35880        NA           NA      NA         NA
82      32321196  45060990  12.00 296.97387        NA           NA      NA         NA
151           NA  45060990  20.00        NA 199.66667           NA      NA         NA
91      32321197  45060990  24.00 248.00367        NA           NA      NA         NA
101     32321198  45060990  48.00 172.95689        NA           NA      NA         NA
112     32321199  45060990  72.00 120.61952        NA           NA      NA         NA
121     32321200  45060990  96.00  84.11963        NA           NA      NA         NA
131     32321201  45060990 120.00  58.66473        NA           NA      NA         NA
141     32321202  45060990 168.00  28.53229        NA           NA      NA   28.53229
161           NA  45060990 696.00        NA        NA 1.027758e-02      NA         NA
```

**Comment by jhhughes256** [comment]
Linking to #118.

**Comment by jhhughes256** [comment]
This issue was addressed as a part of e681d9c. Please see #118 for closing message.
